BeatSaberHTTPStatus/BeatSaberHTTPStatusPlugin.csproj

```diff
@@ -12,8 +12,8 @@
     <TargetFrameworkVersion>v4.6.1</TargetFrameworkVersion>
     <FileAlignment>512</FileAlignment>
     <TargetFrameworkProfile />
-    <Version>1.7.0</Version>
-    <BeatSaberVersion>1.1.0</BeatSaberVersion>
+    <Version>1.8.0</Version>
```

Comment:
Do not change the plugin version. The plugin version gets bumped in the release commit. Until then the version stays the same.